### PR TITLE
doc: add lisa case training data.

### PR DESCRIPTION
**Pull request by squirrelsc**
It generate a jsonl file for llm training purpose. The input is test case descriptions of a test file, output is the test case file content.

### CODE REVIEW

File: docs/tools/finetune_gen.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""
+Update test_spec.rst with test metadata
```
Comment (LiliDeng) [inline docs/tools/finetune_gen.py]: update comments

File: docs/tools/finetune_gen.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+from pathlib import Path
+from typing import Dict
+
+from .doc_generator import TESTS, ClassVisitor, FuncVisitor, extract_metadata, load_path
+
+base_path = Path(__file__).parent
+file_path = (base_path / "../_build/html/finetune_cases.jsonl").resolve()
+system_prompt = "You are a helpful assistant to write Python code for linux validation."
+user_prompt = (
+    "Please bases on the test case description to write a LISA test case.\n\n\n"
```
Comment (LiliDeng) [inline docs/tools/finetune_gen.py]: `Please write a LISA test case based on the test case description.`

File: docs/tools/finetune_gen.py (hunk 1 above)
Comment (squirrelsc) [reply docs/tools/finetune_gen.py]: fixed

File: docs/tools/finetune_gen.py (hunk 2 above)
Comment (squirrelsc) [reply docs/tools/finetune_gen.py]: fixed

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**